tests/test_errors.py

```diff
@@ -21,3 +24,18 @@ def foo():
 """)
     with pytest.raises(ValueError):
         v.get_unused_code(min_confidence=150)
+
+
+def test_non_utf8_encoding(v):
+    code = """\
+def foo():
+    pass
+
+foo()
+"""
+    with tempfile.NamedTemporaryFile(mode='wb', suffix='.py') as f:
```

Comment:
I guess appveyor doesn't allow writing to the global tmp dir. Let's use `dir=REPO`.

Reply:
The tests were failing with the same error message even when we had `dir=REPO`. It is a Windows problem I guess, see https://stackoverflow.com/questions/17502704/python-tempfile-how-to-open-several-times-the-temp-file#17502778

Reply:
Edit: When we had `dir=tests`.

Reply:
Hmm, strange. But I'm confident you'll figure it out :-) Maybe you can try creating the file with a simple `with open(...) as f:` call and see if that works.

Reply:
Python docs say:

> Whether the name can be used to open the file a second time, while the named temporary file is still open, varies across platforms (it can be so used on Unix; it cannot on Windows NT or later).

So yes, we should just use plain old `open` instead - We aren't doing anything fancy with `tempfile` anyways. The only additional step would be to delete the file once we are done.